2.0/TextureLinearInterpolationTest/README.md

```diff
@@ -13,6 +13,8 @@ This asset tests that linear texture interpolation is performed on linear values
 * The second sphere samples its color as an interpolated value from a `2x1` texture. When interpolation happens after sRGB decoding, the final emissive value should also be about `0.5` green. An example of incorrect interpolation is shown below.
 
   ![incorrect](screenshot/incorrect.png)
+  
+> ⚠️ _**WARNING**: For technical and historical reasons, some engines — particularly those using WebGL 1.0 — apply sRGB decoding in the fragment shader. Such engines will "fail" this test, which is designed specifically to highlight a difference that is generally quite subtle. Engines are encouraged to use features like [EXT_sRGB](https://developer.mozilla.org/en-US/docs/Web/API/EXT_sRGB) (or equivalent) moving forward._
```

Comment:
Consider replacing the `EXT_sRGB` reference with "WebGL 2.0 or WebGPU".

In some cases, the WebGL 1.0 extension would result in the same issue as decoding sRGB in a fragment shader. Moreover, that extension may be incompatible with `generateMipmap()` operation.

Reply:
Updated 👍